src/coreclr/src/jit/compiler.cpp

```diff
@@ -5991,11 +6075,11 @@ int Compiler::compCompileHelper(CORINFO_MODULE_HANDLE            classPtr,
     }
 
 #ifdef DEBUG
-    if (JitConfig.DumpJittedMethods() == 1 && !compIsForInlining())
+    if ((JitConfig.DumpJittedMethods() == 1) && !compIsForInlining())
     {
-        printf("Compiling %4d %s::%s, IL size = %u, hash=%08x %s\n", Compiler::jitTotalMethodCompiled,
+        printf("Compiling %4d %s::%s, IL size = %u, hash=0x%08x (%u) %s%s\n", Compiler::jitTotalMethodCompiled,
                info.compClassName, info.compMethodName, info.compILCodeSize, info.compMethodHash(),
-               compGetTieringName());
+               info.compMethodHash(), compGetTieringName(), compGetStressMessage());
```

Comment:
I'm curious why you're printing out the method hash in both hex and decimal.

Reply:
No good reason. I should remove the hex.

The range parser parses its list entries as decimal, so the hex value isn't super useful.

Reply:
COMPLUS_JitHashDump and COMPLUS_NgenHashDump both take hex values to select the method to dump.

Reply:
Right, all the CONFIG_INTEGERs parse as hex. There are a few others...

If we want to standardize on this (which is fine by me) we should update the range parser to also parse hex.

I find it a less intuitive to do manual range isolation in hex, but probably would get used to it pretty quickly.

Reply:
I prefer seeing the method hashes as 8 digit hex values.

Reply:
Let's standardize on hex then... I will have to update the range parser and fix a few other places.

@CarolEidt ok by you?

Reply:
Fine by me; I don't really care whether it's hex or decimal, but agree that consistency is good :-)